code/game/jobs/job/engineering.dm

```diff
@@ -21,6 +21,7 @@
 	exp_requirements = 1200
 	exp_type = EXP_TYPE_ENGINEERING
 	outfit = /datum/outfit/job/chief_engineer
+	important_information = "This role requires you to coordinate a large department. You are required to be familiar with Standard Operating Procedure (Department), basic job duties, and must act in a professional and reasonable manner (roleplay)."
```

Comment:
@Shadeykins Did you mean for me to replace `(Department)` with that department's SOP?

Reply:
yes